README.md

```diff
@@ -30,32 +35,37 @@ git clone git@github.com:bruce3557/NTHUOJ_web.git
 * django-bootstrap
 * django-bootstrap-form
 * requests
+* git
 
 ###Installation:
 * Install dependencies (for ubuntu)
 ```
-apt-get install mysql-server
-apt-get install python-pip
-apt-get install python-mysqldb
-apt-get install nodejs
-apt-get install npm
-npm install -g bower
-ln -s /usr/bin/nodejs /usr/bin/node
-pip install django==1.7
-pip install django-bower
-pip install django-axes
-pip install django-autocomplete-light
-pip install django-datetime-widget
-pip install django-bootstrap
-pip install django-bootstrap-form
-pip install requests
+sudo apt-get install mysql-server
+sudo apt-get install python-pip
+sudo apt-get install python-mysqldb
+sudo apt-get install nodejs
+sudo apt-get install npm
+sudo npm install -g bower
+sudo ln -s /usr/bin/nodejs /usr/bin/node
+sudo pip install django==1.7
+sudo pip install django-bower
+sudo pip install django-axes
```

Comment:
Both is ok for me.
Use ```requirements.txt``` in this PR or in another PR.
It's up to @bbiiggppiigg and @henryyang42